server/cmd/mmctl/commands/extract_e2e_test.go

```diff
@@ -156,27 +160,31 @@ func (s *MmctlE2ETestSuite) TestExtractJobListCmdF() {
 		cmd.Flags().Int("per-page", perPage, "")
 		cmd.Flags().Bool("all", false, "")
 
-		_, appErr := s.th.App.CreateJob(&model.Job{
+		_, appErr := s.th.App.CreateJob(ctx, &model.Job{
 			Type: model.JobTypeExtractContent,
 			Data: map[string]string{},
 		})
 		s.Require().Nil(appErr)
 
 		time.Sleep(time.Millisecond)
 
-		job2, appErr := s.th.App.CreateJob(&model.Job{
+		job2, appErr := s.th.App.CreateJob(ctx, &model.Job{
 			Type: model.JobTypeExtractContent,
 			Data: map[string]string{},
 		})
 		s.Require().Nil(appErr)
+		job2.Logger = nil
```

Comment:
It's not clear to me why the logger is made nil here.  Don't we always want a non-nil logger there?  If we want to suppress output we can have a simple target that doesn't output anywhere.

Reply:
The problem is that the jobs returned by `printer.GetLines()` in https://github.com/mattermost/mattermost/blob/bd6726460a681a3c9a6546003b8893a514b72a63/server/cmd/mmctl/commands/extract_e2e_test.go#L193-L194 don't have a `Logger`. Hence, the `Equal` check fails.

So, instead of assigning them to a variable, calling `InitLogger` for them and then calling `Equal`, I decided to `nil` the logger here.

Do you prefer the other approach?